libpf/interpretertype.go

```diff
@@ -54,22 +54,26 @@ func (i InterpreterType) Frame() FrameType {
 var interpreterTypeToString = map[InterpreterType]string{
 	UnknownInterp: "unknown",
 	PHP:           "php",
-	PHPJIT:        "phpjit",
-	Python:        "python",
-	Native:        "native",
-	Kernel:        "kernel",
-	HotSpot:       "jvm",
-	Ruby:          "ruby",
-	Perl:          "perl",
-	V8:            "v8",
-	Dotnet:        "dotnet",
-	APMInt:        "apm-integration",
+	// OTel SemConv does not differentiate between jitted code and interpreted code.
+	PHPJIT:  "php",
+	Python:  "cpython",
+	Native:  "native",
+	Kernel:  "kernel",
+	HotSpot: "jvm",
+	Ruby:    "ruby",
+	Perl:    "perl",
+	V8:      "v8js",
+	Dotnet:  "dotnet",
+	APMInt:  "apm-integration",
 }
 
 var stringToInterpreterType = make(map[string]InterpreterType, len(interpreterTypeToString))
 
 func init() {
 	for k, v := range interpreterTypeToString {
+		if k == PHPJIT || k == APMInt {
+			continue
+		}
```

Comment:
I'd remove this and keep the symmetry so as not to make assumptions about usage. `PHPJIT` and `APMInt` are valid interpreter types (regardless of how they're used in other packages) so we shouldn't be assuming that `InterpreterTypeFromString` will never be called on them.